File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string res = "";
        int i = 0, j = 0, k = 0;
        while (i < word1.size() || j < word2.size())
        {
            if (i < word1.size())
                res += word1[i++];
            if (j < word2.size())
                res += word2[j++];
        }
        return res;
    }
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution(object):
    def mergeAlternately(self, word1, word2):
        """
        :type word1: str
        :type word2: str
        :rtype: str
        """
        i, res = 0, ""
        while i < len(word1) or i < len(word2):
            if i < len(word1):
                res += word1[i]
            if i < len(word2):
                res += word2[i]
            i+=1
        return res
        